modeling/meters.md

```diff
@@ -0,0 +1,132 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+Meters
+======
+
+```{note}
+This feature is new in Brick v1.3
+```
+
+Brick provides a basic but flexible model for describing meters, submeter hierarchies, and their relationships to the building.
+Meters are equipment that measure the consumption or production of energy, steam, gas or other "substances" in the building.
```

Comment:
We can be more up front about how to use Meters as equipment by providing example points (Electricity_Usage_Sensor, Power_Sensor, etc.) possibly associated with meters. People get easily confused why meters are equipment.

Reply:
Added some more text to clarify